src/zenml/cli/utils.py

```diff
@@ -181,7 +181,7 @@ def print_stack_configuration(
     )
     rich_table.add_column("COMPONENT_TYPE")
     rich_table.add_column("COMPONENT_NAME")
-    items = {typ.value: name for typ, name in config.items()}
+    items = [[typ.value, name] for typ, name in config.items()]
```

Comment:
my version of it was this: ```    items = ([typ.value, name] for typ, name in config.items())
    for item in items:
        rich_table.add_row(*item)```

I think you can remove the `*list(item)` part too.

Reply:
Even better. Thank you. I've updated it with how you did it.